dapps/profile.vue/src/components/verifications/overview/overview.ts

```diff
@@ -60,16 +60,16 @@ export default class VerificationsOverviewComponent extends mixins(EvanComponent
     const runtime = (<any> this).getRuntime();
 
     // use url address or use runtime activeAccount as default
-    this.address = (this as any).$route.params.address || runtime.activeAccount;
+    this.address = (this as any).$route.params.address || runtime.activeIdentity;
 
     // switch issue account
-    this.canIssue = runtime.activeAccount === '0x662fD340606B6c00C51d1915A9f66C081E412e4B';
+    this.canIssue = runtime.activeIdentity === '0x662fD340606B6c00C51d1915A9f66C081E412e4B';
```

Comment:
I thought the identity is not the same as the accountId. Therefore we should check what's the correct issuer identity, right?

Reply:
Should be `0x4F7Fb22489Dd72BBfbBCBac7C68cb499A56309f9` :)